From fe2e4a73480bf2ac2e566052ea682817dddaf61f Mon Sep 17 00:00:00 2001
From: Johannes Demel <demel@uni-bremen.de>
Date: Sat, 14 Jan 2023 18:35:59 +0100
Subject: [PATCH 1/5] ci: Remove license check

This check fails because new contributors don't need to re-license. All
new contributions must be made under LGPL. Thus, this check is
superfluous. Also, it would add an administrative burden that we should
avoid.

Signed-off-by: Johannes Demel <demel@uni-bremen.de>
---
 CMakeLists.txt | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3816757..a6e4f87 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -383,21 +383,6 @@ if(ENABLE_MODTOOL)
   add_subdirectory(python/volk_modtool)
 endif()
 
-########################################################################
-# And the LGPL license check
-########################################################################
-# detect default for LGPL
-if(VERSION_INFO_MAJOR_VERSION GREATER_EQUAL 3)
-  OPTION(ENABLE_LGPL "Enable testing for LGPL" ON)
-else()
-  OPTION(ENABLE_LGPL "Enable testing for LGPL" OFF)
-endif()
-
-if(ENABLE_TESTING AND ENABLE_LGPL)
-  message(STATUS "Checking for LGPL resubmission enabled")
-  add_test("check_lgpl" ${CMAKE_SOURCE_DIR}/scripts/licensing/count_contrib.sh ${CMAKE_SOURCE_DIR}/AUTHORS_RESUBMITTING_UNDER_LGPL_LICENSE.md)
-endif()
-
 ########################################################################
 # Print summary
 ########################################################################
-- 
2.39.2

